microsoft-edge/webview2/concepts/webview-features-flags.md

```diff
@@ -114,6 +114,7 @@ The following are some of the flags we've seen used.
 | `msEdgeDesignerDriverFix ` |  Use this flag to disable getting content and automatically showing the Designer feature. This feature is not supported in WebView2. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
 | `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging.  See [Remote debugging WebView2 WinUI 2 (UWP) apps](../how-to/remote-debugging.md), [Remotely debug Windows devices](../../devtools-guide-chromium/remote-debugging/windows.md). |
+| `msEdgeWebViewApplyWebResourceRequestedFilterForOOPIFs` | If this flag is enabled, `AddWebResourceRequestedFilter` will also apply to out-of-process iframes. <br/>.NET: [AddWebResourceRequestedFilter](/dotnet/api/microsoft.web.webview2.core.corewebview2.addwebresourcerequestedfilter) <br/>WinRT: [AddWebResourceRequestedFilter](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2#addwebresourcerequestedfilter) <br/>Win32: [AddWebResourceRequestedFilterWithRequestSourceKinds](/microsoft-edge/webview2/reference/win32/icorewebview2_22#addwebresourcerequestedfilterwithrequestsourcekinds) |
```

Comment:
does this row apply to both of the overloads in .NET & WinRT?
I only linked to one of the two overloads of this method name.

Reply:
result:
```
If this flag is enabled, the `AddWebResourceRequestedFilter` method 
(including overloads) will also apply to out-of-process iframes.
```
@monica-ch @peiche-jessica 
Closing Comment.